cypress-tests/cypress/support/commands.js

```diff
@@ -25,7 +25,7 @@
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 
 // commands.js or your custom support file
-import ConnectorAuthDetails from "../../../.github/secrets/creds.json";
```

Comment:
revert

Reply:
@pixincreate We can add the path as env then everyone would just export the path instead of updating it here!
We can take up that change in this pr may be?

Reply:
done

Reply:
@likhinbopanna is the change mentioned above is done?

Reply:
i have just reverted the connectorAuthDetails and that is done

Reply:
I'll take this up and update it on my [PR](https://github.com/juspay/hyperswitch/pull/4396)...